### ISSUE TITLE
0.8.0 requires "resolver" due to Cargo bug

### ISSUE THREAD

**Issue by adamreichold** [tkaitchuck/aHash#127]
[The `Cargo.toml` published with 0.8.0](https://docs.rs/crate/ahash/0.8.0/source/Cargo.toml) contains the "resolver" key which is not present in [the original `Cargo.toml`](https://docs.rs/crate/ahash/0.8.0/source/Cargo.toml.orig) suggesting that the package was published using a Cargo version affected by https://github.com/rust-lang/cargo/issues/10954

Could you publish a semver-compatible release that does not contain the "resolver" key? This would allow the `rust-numpy` crate to update its dependency on `ahash` without changing MSRV, c.f. https://github.com/PyO3/rust-numpy/actions/runs/3073552488/jobs/4965760601

**Comment by adamreichold** [comment]
Friendly ping. Just reuploading with a bumped version and using a current Cargo would really help. Thanks!

**Comment by tkaitchuck** [comment]
I am working on 0.8.1 which will also have some other improvements.

**Comment by tkaitchuck** [comment]
I just released 0.8.1, but it has the same problem. Is this caused because I released using nightly?

**Comment by adamreichold** [comment]
> I just released 0.8.1, but it has the same problem. Is this caused because I released using nightly?

It should not be the case when the current nightly is used, only if you have pinned the version to one still affected by the above mentioned bug.

**Comment by tkaitchuck** [comment]
This appears to have been fixed with the 0.8.2 release. I will release using stable from now on as a precaution. Closing for now.

**Comment by adamreichold** [comment]
Thanks, the resolve issue is now fixed indeed.

Sadly, we cannot bump to 0.8.x as the `atomic-polyfill` feature shadows a dependency which the Rust 1.48 toolchain that is shipped by Debian does not yet support, so we'll have to stick to 0.7.x for now after all, c.f. https://github.com/PyO3/rust-numpy/actions/runs/3442951690/jobs/5744015218.

If I could have a wish, adding the `rust-version` key to this crate's `Cargo.toml` would be nice. (I would make a PR but I did not find MSRV information.)
